src/mui/field/ImageField.js

```diff
@@ -11,6 +11,10 @@ const styles = {
     },
 };
 
+const clear = {
```

Comment:
@jpetitcolas At this time, I wonder if we are targeting the right problem. I suppose the [float; left](https://github.com/marmelab/admin-on-rest/blob/master/src/mui/field/ImageField.js#L6) was used in correlation with the `ImageInput` in case of multiple file upload, to display "in a row" images.

But I don't think it's the right way for `ImageField` to define how images must be displayed. I would delegate this to the user by removing this `float: left` property and find a way to let user define how `ImageField` must be displayed.

What do you think about this ?